cypress/support/pages/settings.pageObject.js

```diff
@@ -0,0 +1,69 @@
+import PageObject from '../PageObject';
+
+class settingsPageObject extends PageObject {
+  url = '/settings';
+
+  loggedUser() {
+    cy.login();
+  }
+
+  updateUsername(username) {
+    cy.getByDataCy('settings-username')
+      .clear()
+      .type(username);
+  }
+
+  updateBio(bio) {
+    cy.getByDataCy('settings-bio')
+      .type(bio);
+  }
+
+  updateEmail(email) {
+    cy.getByDataCy('settings-email')
+      .clear()
+      .type(email);
+  }
+
+  updatePassword(password) {
+    cy.getByDataCy('settings-password')
+      .type(password);
+  }
+
+  submitButton() {
+    cy.getByDataCy('settings-submit')
+      .click();
+  }
+
+  assertBio(bio) {
+    cy.getByDataCy('settings-bio')
+      .should('contain', bio);
+  }
+
+  logOut() {
+    cy.getByDataCy('/settings')
+      .click();
+    cy.get('.btn-outline-danger')
+      .click();
+    cy.wait(1500);
```

Comment:
You almost never need to wait for an arbitrary period of time. There are always better ways to express this in Cypress.